steps_log.txt

```diff
@@ -8,4 +8,6 @@ In DeepLearning:
     6 - A lot of the obtained values are quite similar, despite wildly different models (MLP with a 50 neurons layer vs MLP with a 500 neurons layer followed by another 100 neurons layer had practically the same score). This lead me to believe my model wasn't actually doing much. After googling for help, I realized I had forgotten to normalize the dataset. Also, our dataset is really small with only 1k examples, thus, I ought to increase our learning rate parameter.
 
     7 - Decided model was working as well as it could with the available data. Went to Kaggle to look for new, different data -> dataset with a bunch of pictures of indian dishes (objective is to identify which dish is which).
-    8 - Just like in the previous problem, it's a multiclass classification problem, but now with computer vision. For images and visual data, Convolutional Neural Networks (CNN) work best. Problem is, sklearn doesn't have CNNs, so I found this guide on how to allow tensorflow to work on ARM chips -> https://towardsdatascience.com/setting-up-apples-new-m1-macbooks-for-machine-learning-f9c6d67d2c0f
\ No newline at end of file
+    8 - Just like in the previous problem, it's a multiclass classification problem, but now with computer vision. For images and visual data, Convolutional Neural Networks (CNN) work best. Problem is, sklearn doesn't have CNNs, so I found this guide on how to allow tensorflow to work on ARM chips -> https://towardsdatascience.com/setting-up-apples-new-m1-macbooks-for-machine-learning-f9c6d67d2c0f
+    9 - Looked for a way to deal with png files with python -> https://machinelearningmastery.com/how-to-load-and-manipulate-images-for-deep-learning-in-python-with-pil-pillow/
+    10 - After trying to use the full dataset, PC starts slowing down due to lack of memory. Solution to big datasets -> use batches https://medium.com/@mrgarg.rajat/training-on-large-datasets-that-dont-fit-in-memory-in-keras-60a974785d71
```

Comment:
Please add a new line at the end of file in order to avoid diffs that display changes on previous lines because of the added \n character